fitbenchmarking/cli/main.py

```diff
@@ -0,0 +1,109 @@
+"""
+This is the main entry point into the FitBenchmarking software package.
+For more information on usage type fitbenchmarking --help 
+or for more general information, see the online docs at
+docs.fitbenchmarking.com.
+"""
+
+from __future__ import (absolute_import, division, print_function)
+import argparse
+import glob
+import os
+import sys
+
+from fitbenchmarking.fitting_benchmarking import fitbenchmark_group
+from fitbenchmarking.results_output import save_results_tables
+from fitbenchmarking.utils.options import Options
+
+
+def get_parser():
+    """
+    Creates and returns a parser for the args.
+    """
+
+    epilog = '''Usage Examples:
+
+    $ fitbenchmarking benchmark_problems/NIST/*
```

Comment:
change `fitbenchmarking benchmark_problems/NIST/*` to change `fitbenchmarking examples/benchmark_problems/NIST/*` and the same below

Reply:
done